### ISSUE TITLE
TS 3.9+ missing `registerProperty` method in `CSS`

### ISSUE THREAD

**Issue by daneden** [microsoft/TypeScript#38593]
**TypeScript Version:**  both 3.9.2 and 4.0.0-dev20200514

**Search Terms:** `registerProperty`

**Code**

```ts
interface CSS {
  registerProperty: (any) => any
}

CSS?.registerProperty({
  name: "--line-direction",
  syntax: "trbl | tlbr | center",
  initialValue: "tlbr",
  inherits: true,
})
```

**Expected behavior:**
- Code compiles successfully

**Actual behavior:**
- TypeScript throws an error: “Property `registerProperty` does not exist on type `typeof CSS`.”

**Playground Link:**
- [v3.9.2 Playground Link (With Errors)](http://www.staging-typescript.org/play?ts=3.9.2&ssl=1&ssc=1&pln=10&pc=3#code/JYOwLgpgTgZghgYwgAgMIGV3IN4ChnJQQDmwAzpFAApQD2ADtGAJ4BcyAFPXFHALZl2cEMwCUyALwA+ZMOa4AvrlwZ0AfgB0RUhWg0GTZhzwEQ-COwBEAWmsAbUBGsATYEQRhgtEJYA0+ZDJmcDgAD3YAbUswKGsAIzs-ZGi7eKgkyyRwaEsAXQ0AK1pQDktkAB9k0X8CUGBPODsANUaAVwtksDs49JrkUAALaHrBZBj2-wVRIA)
- [v3.8.3 Playground Link (No Errors)](http://www.staging-typescript.org/play?ts=3.8.3#code/JYOwLgpgTgZghgYwgAgMIGV3IN4ChnJQQDmwAzpFAApQD2ADtGAJ4BcyAFPXFHALZl2cEMwCUyALwA+ZMOa4AvrlwZ0AfgB0RUhWg0GTZhzwEQ-COwBEAWmsAbUBGsATYEQRhgtEJYA0+ZDJmcDgAD3YAbUswKGsAIzs-ZGi7eKgkyyRwaEsAXQ0AK1pQDktkAB9k0X8CUGBPODsANUaAVwtksDs49JrkUAALaHrBZBj2-wVRIA)

**Related Issues:** 
- #38554

**Comment by BarnT** [comment]
Also `CSS.paintWorklet.addModule()` throws error:
"_Property 'paintWorklet' does not exist on type 'typeof CSS'._"

**Comment by kongxiangyan** [comment]
> ```ts
> interface CSS {
>   registerProperty: (any) => any
> }
> ```

I am encountering the same problem, if you hadn't solve it, use `namespace` instead of `interface`. 

Here is my code, all definition is refer to [`CSS.registerProperty()` - MDN](https://developer.mozilla.org/en-US/docs/Web/API/CSS/RegisterProperty)

```ts
declare namespace CSS {
  interface PropertyDefinition {
    name: string
    syntax?: string
    inherits: boolean
    initialValue?: string
  }
  function registerProperty (propertyDefinition: PropertyDefinition): undefined
}
```

**Comment by scriptex** [comment]
> Also `CSS.paintWorklet.addModule()` throws error: "_Property 'paintWorklet' does not exist on type 'typeof CSS'._"

I fixed this with the following code:

```ts
declare namespace CSS {
	namespace paintWorklet {
		export function addModule(url: URL): void;
	}
}
```

**Comment by jakebailey** [comment]
This was fixed in #53133, which restored CSS as a namespace.
